### PR TITLE
Codechef STONES Solution in C

### PR DESCRIPTION
**Pull request by ritish099**
**Fixes issue:#4822**

**Changes:**

Added Codechef STONES Solution in C
Added README for Codechef STONES Problem

### CODE REVIEW

**Comment by ritish099** [comment]
Sir @arnavb,  Check it out

**Comment by shivani789** [comment]
Please assign me this issue for c++ language

File: code/online_challenges/src/codechef/STONES/STONES.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#include <string.h>
+
+int main(void) 
+{
+    int t;
+    scanf("%d", & t);
+    while (t--) 
+    {
+        char s[100], r[100];
+        int count = 0, i, j;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STONES/STONES.c]: Declare i and j right above their respective loops.

File: code/online_challenges/src/codechef/STONES/STONES.c [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        scanf("%s", s);
+        scanf("%s", r);
+        for (j = 0; j < strlen(r); ++j)
```
Comment (arnavb) [inline code/online_challenges/src/codechef/STONES/STONES.c]: This is extremely inefficient. `strlen` computes the length of the string every iteration. Store the length in a variable.

**Pull request merged**